Apps/phdigitalshadows/dsapi/config/__init__.py

```diff
@@ -1,7 +1,6 @@
-#
-# Copyright (c) 2017 Digital Shadows Ltd.
+# File: __init__.py
```

Comment:
Can you please clarify which folder you mean to add here as I am not sure I understand the comment?

Reply:
Fixed
@somya-metron
I think it is about adding the subfolder to which the init file belongs. for instance, in the above-mentioned case, it could be File: dsapi/config/__init__.py

Reply:
Great! Thanks for the help @amangal-crest